packages/tpetra/core/test/CrsMatrix/CrsMatrix_NodeConversion.cpp

```diff
@@ -198,8 +198,9 @@ namespace {
     TEUCHOS_UNIT_TEST_TEMPLATE_1_INSTANT( CrsMatrix, NodeConversion, N2 )
 
   TPETRA_ETI_MANGLING_TYPEDEFS()
-
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
```

Comment:
We also need to protect the declaration of the test with this macro.  Otherwise, the compiler may emit "you declared a class without using it" warnings, and may also emit "you're using deprecated code" warnings.

Reply:
We routinely declare unused test macros and the compiler never complains...